specs/eip4844/p2p-interface.md

```diff
@@ -188,16 +215,22 @@ Request Content:
 ```
 
 Response Content:
+
+```python
+class BlobSidecars(Container):
+  block_root: Root
+  List[BlobSidecar, MAX_BLOBS_PER_BLOCK]
+
 ```
 (
-  List[BlobsSidecar, MAX_REQUEST_BLOBS_SIDECARS]
+  List[BlobSidecars, MAX_REQUEST_BLOCKS_EIP4844]
```

Comment:
Using individual blobs here is not ideal, I wonder if we can have a wrapper that contains all the blobs in the slot

Reply:
that's what `BlobSidecars` is. The plural version. I suggest we make it more clear `AggregateBlobSidecar` or something

Reply:
Ah. I missed the plural version. Makes sense to call it `Aggregate` to keep it somewhat consistent with attestations and sync messages

Reply:
`Aggregate` though implies we compressed some information - I'll leave the naming question open for a bit, pending a name that captures the list-but-not-aggregate aspect.

Reply:
`BlockBlobSidecars` maybe. 

Or indeed, an `AggregateBlobsSidecar` with an aggregate proof and different field organisation, though I'm not convinced it's worth the complexity (vs simply reusing the existing "shapes").

Reply:
yeah, I agree with "aggregate" having a semantic collision with Aggregate attestations...

Still need a better name than the subtle plural. Maybe `FullBlobSidecar` or something.

Reply:
discussed elsewhere, resolving this convo as dupe